app/common/common-module.js

```diff
@@ -41,6 +41,7 @@ angular.module('ushahidi.common', [
 .controller('navigation', require('./controllers/navigation.js'))
 .controller('PageMetadata', require('./controllers/page-metadata.js'))
 .controller('notifier', require('./controllers/notifier.js'))
+.controller('intercom', require('./controllers/intercom.js'))
```

Comment:
So I'd be tempted to take this out of Angular entirely - if someone loads a deployment and hits a major error then angular might never bootstrap.. 
If we load intercom outside angular - then they at least get a support channel directly on the failure page. Does that make sense?